virtualenv.py

```diff
@@ -354,11 +356,11 @@ def copyfile(src, dest, symlink=True):
         logger.info('Copying to %s', dest)
         copyfileordir(src, dest, symlink)
 
-def writefile(dest, content, overwrite=True):
+def writefile(dest, content, overwrite=True, use_fsenc=False):
     if not os.path.exists(dest):
         logger.info('Writing %s', dest)
         with open(dest, 'wb') as f:
-            f.write(content.encode('utf-8'))
+            f.write(content.encode(sys.getfilesystemencoding() if use_fsenc else 'utf-8'))
```

Comment:
Here's a problem - ```sys.stdout.encoding``` is ```None``` if the terminal is not interactive (python invoked via ```subprocess```, etc.). From testing results ```sys.getfilesystemencoding()``` is OK so I keep them.